docs/using-the-jdbc-driver/RdsProxy.md

```diff
@@ -0,0 +1,24 @@
+# RDS Proxy FAQ
+
+## What is Amazon RDS Proxy
+Amazon RDS Proxy is a fully managed, highly available database proxy for Amazon Relational Database Service (RDS). Similar to the AWS Advanced JDBC Driver, Amazon RDS Proxy also supports failover and is able to route requests to new database instances to reduce downtime.
+
+For more information, see the [[Amazon RDS Proxy FAQs](https://aws.amazon.com/rds/proxy/faqs/)].
+
+## What is the difference between Amazon RDS Proxy and AWS Advanced JDBC Driver
```

Comment:
Another huge difference. Even if user app uses RDS Proxy, it still needs extra code to handle network outages and handle them properly. AWS Wrapper already has such logic built-in.